editor/views/rai.py

```diff
@@ -0,0 +1,262 @@
+                value=metadata.data_imputation_protocol,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_IMPUTATION_PROTOCOL, metadata, key),
+            )
+
+    with col2.expander("**Data uses and social impact**", expanded=True):
+        # Data use cases
+        oneToManyProperty(
+            "**Use cases**",
+            metadata,
+            metadata.data_use_cases,
+            "metadata-data-use-cases_",
+            "Dataset use case - training, testing, validation, development or production use, fine tuning, others (please specify), usage guidelines, recommended uses, etc.",
+            RaiEvent.RAI_DATA_USE_CASES,
+        )
+
+        # Data biases
+        oneToManyProperty(
+            "**Data biases**",
+            metadata,
+            metadata.data_biases,
+            "metadata-data-biases_",
+            "**Data biases**. Involves understanding the potential risks associated  with data usage and to prevent unintended and potentially harmful consequences that may arise from using models trained on or evaluated with the respective data",
+            RaiEvent.RAI_DATA_BIAS,
+        )
+
+        # Data sensitive
+        oneToManyProperty(
+            "**Personal and sensitive information**",
+            metadata,
+            metadata.personal_sensitive_information,
+            "metadata-personal-sensitive-information_",
+            "Personal and sensitive information, if contained within the dataset, can play an important role in the mitigation of any risks and the responsible use of the datasets",
+            RaiEvent.RAI_SENSITIVE,
+        )
+
+        key = "metadata-social-impact"
+        st.text_area(
+            label=("**Social impact**. Discussion of social impact, if applicable"),
+            key=key,
+            value=metadata.data_social_impact,
+            on_change=handle_rai_change,
+            args=(RaiEvent.RAI_DATA_SOCIAL_IMPACT, metadata, key),
+        )
+
+        # Data Limitations
+        oneToManyProperty(
+            "**Data limitations**",
+            metadata,
+            metadata.data_limitations,
+            "metadata-data-limitations_",
+            "Known limitations - Data generalization limits (e.g related to data distribution, data quality issues, or data sources) and on-recommended uses.",
+            RaiEvent.RAI_DATA_LIMITATION,
+        )
+
+        key = "metadata-data-maintenance"
+        st.text_area(
+            label=(
+                "**Data release maintenance**. Versioning information in terms of the updating timeframe, the maintainers, and the deprecation policies. "
+            ),
+            key=key,
+            value=metadata.data_release_maintenance_plan,
+            on_change=handle_rai_change,
+            args=(RaiEvent.RAI_MAINTENANCE, metadata, key),
+        )
+
+
+def oneToManyProperty(
+    wrapperTitle: str, metadata: Metadata, attribute, key: str, label: str, event: str
+):
```

Comment:
Can you add a docstring to specify the behaviour? Especially for `attribute`, it's tricky to understand when it should be a list or not (and why)

Reply:
added